rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java

```diff
@@ -48,13 +48,18 @@ private InformixUtil() {
     public static final String LOCK_OVF_SQL = "select coalesce(cast(value as int)) as value from sysprofile where name = 'ovlock'";
     public static final String TRANSACTION_OVF_SQL = "select coalesce(cast(value as int)) as value from sysprofile where name = 'ovtrans'";
     public static final String USER_OVF_SQL = "select coalesce(cast(value as int)) as value from sysprofile where name = 'ovuser'";
-    public static final String DB_SEQ_SCAN_SQL = "select coalesce(cast(seqscans as int))DATABASE_LOCK_TABLE_OVERFLOW,tabname as table_name from sysmaster:sysptprof where dbsname = %s and seqscans >= %d and tabname not like 'sys%%'";
+    public static final String DB_SEQ_SCAN_SQL = "select coalesce(cast(seqscans as int))seqscan,tabname as table_name from sysmaster:sysptprof where dbsname = %s and seqscans >= %d and tabname not like 'sys%%'";
     public static final String DB_SEQ_SCAN_TABLE_SQL = "SELECT count(tabname)  as number_of_tables_having_sequential_scans FROM SYSPTPROF WHERE dbsname = %s and seqscans >= %d and tabname not like 'sys%%';";
 
     //Disk Read & Write
     public static final String DB_DISK_WRITE_COUNT_SQL = "SELECT VALUE FROM SYSPROFILE WHERE NAME = 'dskwrites';";
     public static final String DB_DISK_READ_COUNT_SQL = "SELECT VALUE FROM SYSPROFILE WHERE NAME = 'dskreads';";
 
+    public  static  final String DB_LOCK_WAITS_SQL = "select value from sysprofile where name = 'lockwts'";
+    public static final String DB_CACHE_READ_RATIO_SQL = "select (1-(sp0.value/sp1.value))*100 as res from sysprofile sp0 left join sysprofile sp1 on sp1.name = 'bufreads' where sp0.name= 'dskreads'";
+    public static final String DB_CACHE_WRITE_RATIO_SQL = "select (1-(sp0.value/sp1.value))*100 as res from sysprofile sp0 left join sysprofile sp1 on sp1.name = 'bufwrites' where sp0.name= 'dskwrites'";
```

Comment:
Could you please explain what you are trying to achieve over here? 
join clause does not making much sense to me.

Reply:
This metric is obtained from two different rows with the 'value' column having values 'bufreads' and 'dskreads' from sysprofile table. Hence I had to  join the table with itself. Note: We have an onstat command for this metric and we are not using this query in local monitoring.

Reply:
Same as above.

Reply:
Above one is applicable to the this also.